pkg/plugins/plugins.go

```diff
@@ -49,9 +49,21 @@ func FindPlugin(pluginame string) (string, error) {
 	return "", fmt.Errorf("cannot find plugin in path or %s: %s", pluginDir, cmd)
 }
 
-func GetAllTknPluginFromPaths() []string {
-	pluginlist := []string{}
+// Execute the plugin binary and returns plugin's description
+func getPluginDescription(pluginPath string) string {
+
```

Comment:
I dont like to bikesched about code formatting but why so many newlines between each line of code?

Reply:
okay, I will remove extra line